components/SiteFooter.vue

```diff
@@ -32,9 +34,13 @@
   </v-footer>
 </template>
 
-<style>
+<style scoped>
 a:link,
 a:visited {
   color: white;
+  text-decoration: none;
+}
+.col {
+  padding: 6px !important;
```

Comment:
In the future, we might want to move away from `!important` in CSS, but tonight, let's just get this working.

Reply:
Sorry this PR is full of CSS habits that I know @Ermish won't like either lol 😭

Reply:
😱 😱 😱